etc/behat/services.xml

```diff
@@ -31,6 +32,8 @@
             <argument>%mink.parameters%</argument>
         </service>
 
+        <service id="sylius.behat.converter.country_name_to_code" class="%sylius.behat.converter.country_name_to_code.class%" scope="scenario" public="false" />
```

Comment:
No need for scope scenario as the service and its all dependencies (none) are stateless.